build/simp-environment.spec

```diff
@@ -33,17 +33,14 @@
 
 Summary: The SIMP Environment Scaffold
 Name: simp-environment
-Version: 6.2.10
+Version: 6.3.0
 Release: 0%{?dist}
 License: Apache License 2.0
 Group: Applications/System
 Source: %{name}-%{version}-%{release}.tar.gz
 Buildroot: %{_tmppath}/%{name}-%{version}-%{release}-buildroot
 Requires: libselinux-utils
 Requires: policycoreutils
-Requires: pupmod-simp-simp >= 3.0.0
-Requires: pupmod-simp-pki >= 6.0.0-0
```

Comment:
We probably need to implement the optional dependency checking before ripping this out. The automatic RPM dependencies are only added if you're using the SIMP RPM build system.

https://github.com/simp/pupmod-simp-simplib/pull/148